hadoop-ozone/dist/src/main/smoketest/recon/recon-nssummary.robot

```diff
@@ -131,7 +132,8 @@ Check Recon Namespace Summary Key
     Wait For Summary      ${SUMMARY_URL}?path=/${VOLUME}/${BUCKET}/file1   KEY
 
 Check Recon Namespace Summary Directory
-    Wait For Summary      ${SUMMARY_URL}?path=/${VOLUME}/${BUCKET}/dir1/dir2   DIRECTORY
+    Run Keyword If    '${BUCKET_LAYOUT}' == 'LEGACY'                    Wait For Summary      ${SUMMARY_URL}?path=/${VOLUME}/${BUCKET}/dir1/dir2/   DIRECTORY
```

Comment:
After running the test with the parameter `-v BUCKET_LAYOUT:LEGACY`, here it will execute the part for LEGACY.

Reply:
The trailing slash is necessary. Without it, we get `PATH_NOT_FOUND` and entity `UNKNOWN`.

> the command after this line 135 will only be executed when '${BUCKET_LAYOUT}' == 'LEGACY'.
If bucket layout is FSO, it will run the command line below instead, with the subtle difference of the trailing slash in the directory path.

You are right. That's how it works.

We give `${BUCKET_LAYOUT}` a default value of FSO and [here](https://github.com/xBis7/ozone/blob/HDDS-7121/hadoop-ozone/dist/src/main/compose/ozone-legacy/test.sh#L31) we execute the smoketest with value LEGACY.

Reply:
Thanks @xBis7 for the explanation. The link would now be [this](https://github.com/xBis7/ozone/blob/HDDS-7121/hadoop-ozone/dist/src/main/compose/ozone-legacy-bucket/test.sh#L31) since the dir is renamed.